### ISSUE TITLE
Add <inheritdoc> tag support

### ISSUE THREAD

**Issue by mikernet** [praeclarum/FuGetGallery#126]
Seems that XML comments that use `<inheritdoc>` tags do not show the inherited docs. Would be great to add this.

**Comment by praeclarum** [comment]
Yes! I haven't supported it because it may require downloading additional packages to resolve. But, I should at least make the local case work.
